Playground/src/components/commandDropdownComponent.tsx

```diff
@@ -96,7 +97,7 @@ export class CommandDropdownComponent extends React.Component<ICommandDropdownCo
                                                     Utilities.StoreStringToStore(this.props.tooltip, m.label);
                                                     this.setState({isExpanded: false, activeState: m.label});
                                                 }
-                                            }} title={m.label}>
+                                            }} title={m.tooltip}>

```

Comment:
you can stay defensive here and do `title={m.tooltip || ''}`, but if everything has a tooltip it should be fine.

Reply:
sure! I'll use the label as a fallback actually